Exercises/Exercise-1/main.py

```diff
@@ -12,8 +14,19 @@
 
 
 def main():
-    # your code here
-    pass
+    for uri in download_uris:
```

Comment:
Add parallel download possibility. By default it should download one file after another. But there should be optional flag e.g. "--workers" specifying number of parallel downloads to run.